README.md

```diff
@@ -47,6 +47,22 @@ go build
 
 sudo packer build boards/odroid-u3/archlinuxarm.json
 ```
+## Run in Docker
+This method is primarily for macOS users where is no native way to use qemu-user-static.
+Dockerfile is modeled on [packer-builder-arm-image](https://github.com/solo-io/packer-builder-arm-image).
+#### Build docker image:
```

Comment:
'###' instead of '####'?

Reply:
I can change it, but it's not a typo. It's a subsection of Docker's section. Do you want to have it changed?

Reply:
I am not following :) If
> ## Run in Docker

is a parent section of:
> #### Build docker image

Then it should be one "#" more, thus 3* "#" = "###", right?

Reply:
oh of course you are right! I wrote it late and when I read your comment I did not verify it :)